content/articles/2018-06-conflicted.Rmd

```diff
@@ -0,0 +1,86 @@
+---
+title: "conflicted: a new approach to resolving ambiguity"
+author: Hadley Wickham
+date: '2018-06-02'
+slug: conflicted
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/M03OCZvYSxY
+  author: Vincent van Zalinge
+tags: []
+---
+
+```{r}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+
+We're stoked to announce the initial release of the [conflicted](https://github.com/r-lib/conflicted#conflicted) package. The goal of conflicted is to provide alternative way of resolving conflicts caused by ambiguous function names. R handles ambiguity by reporting conflicts when you load a package, then having the most recently loaded package win. This can make it hard to detect conflicts, because it's easy to miss the messages since you often load packages at the top of the script, and you don't see a problem until much later. conflicted takes a different approach to resolving ambiguity, instead making every conflict an error and forcing you to always choose which function to use.
```

Comment:
* California Hadley leaves his mark with "stoked"... 🏄  (leave it, it's hilarious)
* _an_ alternative way
* _then having_ sounds a little weird to me (it matches with reporting, but... You could try "To handle ambiguity R reports..." or something of the like)
* Swap in explicitly for always?